spec/requests/appropriate_bodies/teachers/index_spec.rb

```diff
@@ -25,6 +25,18 @@
         end
       end
 
+      context "when there are more than 10 claimed ECTs" do
+        it 'displays pagination' do
+          FactoryBot.create_list(:teacher, 11, trs_first_name: "John", trs_last_name: "Smith").each do |teacher|
+            FactoryBot.create(:induction_period, :active, teacher:, appropriate_body:)
+          end
+
+          get("/appropriate-body/teachers")
+
+          expect(response.body).to include('govuk-pagination')
+        end
+      end
```

Comment:
Slightly frustrating I couldn't test this in the view spec because it's pulling in more data from the record - I'll add an issue to improve this.